docs/core/logger.md

```diff
@@ -298,9 +298,10 @@ If you already have an object containing a `message` key and an additional prope
         logger.info('This is a log with an extra variable', { data: myImportantVariable });
         
         // You can also pass multiple parameters
-        logger.info('This is a log with 2 extra variables',
+        logger.info('This is a log with 3 extra objects',
```

Comment:
Question: why did you reintroduced the possibility to add > 2 parameters?

Reply:
It was actually always possible, both prior of #614 and after that. Is there a requirement to limit the number to two? In [your comment](https://github.com/awslabs/aws-lambda-powertools-typescript/issues/565#issuecomment-1048639733) in #565 you give an example where providing more than two objects should be possible.

Reply:
I see, yes that's how I implemented it originally. No requirement or action needed. 
As I was reviewing this PR I saw this change, which confused me on why this was added for the current scope.